tamr_toolbox/notifications/email_info.py

```diff
@@ -0,0 +1,38 @@
+from dataclasses import dataclass
+from tamr_toolbox.models.data_type import JsonDict
+from typing import List
+
+
+@dataclass
+class EmailInfo:
```

Comment:
If using a dataclass this should go in the [models/ directory](https://github.com/Datatamer/tamr-toolbox/tree/develop/tamr_toolbox/models)

Reply:
email_info removed